hub/scribe/service.py

```diff
@@ -776,6 +786,7 @@ def get_controlling(_name):
         activated_at_height = self.db.get_activated_at_height(height)
         activate_in_future = defaultdict(lambda: defaultdict(list))
         future_activations = defaultdict(dict)
+        total_activated = set()
```

Comment:
I don't see this being used in a meaningful way. Also name "total" implies its a numerical value.